Fix ConfiguredName

donavanbecker opened the pull request:
From what I can see `ConfiguredName` should be setup like this.

Also added a log to indicate if the name is empty completely.

Supereg reviewed (dismissed):
Great catch 🚀

hjdhjd reviewed (dismissed):
Only suggestion would be to ensure the verbiage is consistent with other warning/error messages in format/wording (I presume it is?).

donavanbecker commented:
> Only suggestion would be to ensure the verbiage is consistent with other warning/error messages in format/wording (I presume it is?).

I have removed the log and added the validation to the check.
